ayon_api/entity_hub.py

```diff
@@ -2474,6 +2505,7 @@ def from_entity_data(cls, project, entity_hub):
             folder_types=project["folderTypes"],
             task_types=project["taskTypes"],
             name=project["name"],
+            statuses=project["statuses"],
```

Comment:
I think this was a bug on the existing code missing `statuses` for the `ProjectEntity`

Reply:
Yes, looks like the method is not used.